src/coreclr/nativeaot/System.Private.CoreLib/src/System/Reflection/Runtime/TypeParsing/TypeParser.cs

```diff
@@ -1,222 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-
-using System;
-using System.Diagnostics;
-using System.Collections.Generic;
-using System.Reflection.Runtime.Assemblies;
-
-namespace System.Reflection.Runtime.TypeParsing
-{
-    //
-    // Parser for type names passed to GetType() apis.
-    //
-    internal sealed class TypeParser
```

Comment:
The unified implementation evolved from this native aot-specific implementation.